samples/Cosmonaut.NetCore2WebApi/Controllers/SampleController.cs

```diff
@@ -0,0 +1,169 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Cosmonaut.Extensions;
+using Cosmonaut.NetCore2WebApi.Contract;
+using Cosmonaut.NetCore2WebApi.Model;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.Documents.SystemFunctions;
+using Microsoft.EntityFrameworkCore;
+
+namespace Cosmonaut.NetCore2WebApi.Controllers
+{
+    public class SampleController : Controller
+    {
+        private readonly ICosmosStore<Person> _personsCosmosStore;
+
+        public SampleController(ICosmosStore<Person> personsCosmosStore)
+        {
+            _personsCosmosStore = personsCosmosStore;
+        }
+
+        [HttpGet("/api/persons")]
+        public async Task<IActionResult> GetAllPaged([FromQuery] int pageSize, [FromQuery] string continuationToken,
```

Comment:
Can you default `pageSize` to some number or check if it's provided? If it's blank you will get an exception.